.github/workflows/integrations.yml

```diff
@@ -85,6 +88,11 @@ jobs:
         run : |
           python -m pip install external/TraitGen
 
+      - name: Install TraitGen test dependencies
+        run : >
+          python -m pip install -r external/TraitGen/tests/requirements.txt
+          | grep "Requirement already satisfied: openassetio"
```

Comment:
[question] What is the purpose of the grep here?

Reply:
If you can please, otherwise it just look like were filtering output...  So the intention is that this step will error if thats the case and fail the rest of the job - so it is a canary rather than something that actually affects what is installed? Also, have we double checked this actually happens (i.e. is `pipefail` definitely set? We've been burnt by that before now.